core/lib/snapshots_applier/src/lib.rs

```diff
@@ -303,25 +278,27 @@ impl<'a, 'b> SnapshotsApplier<'a, 'b> {
         Ok(())
     }
 
+    #[tracing::instrument(level = "debug", err, skip(self))]
```

Comment:
why this line is needed?

Reply:
It's not strictly needed, but it's useful: It adds context (namely, a [span](https://docs.rs/tracing/latest/tracing/#spans) with `chunk_id` argument) to all logs emitted within the method (and all code that it calls). This allows to not repeat all context explicitly in all logs, so it's useful for debugging etc.